staging/src/k8s.io/client-go/tools/portforward/portforward.go

```diff
@@ -406,6 +406,7 @@ func (pf *PortForwarder) handleConnection(conn net.Conn, port ForwardedPort) {
 	case <-remoteDone:
 	case <-localError:
 	}
+	_ = dataStream.Reset()
```

Comment:
can you add a good comment here about why we're doing this at all (to ensure dataStream drains) and why we're doing it here

Reply:
added.